esp/templates/inclusion/program/class_teacher_list_row.html

```diff
@@ -108,8 +108,8 @@ <h3>Cancellation Request for {{ cls|truncatewords:6 }}</h3>
 
 {% if crmi.allow_coteach %}
 <tr>
-  <td class="clsleft"><b>Teachers (<a href="mailto:{{ cls.emailcode }}-teachers@{{ email_host_sender }}">email</a>): </b></td>
-  <td class="clsmiddle" colspan="5">{% for t in cls.get_teachers %}{% ifnotequal t request.user %}{{ t.first_name }} {{ t.last_name }}{% if not forloop.last %}, {% endif %}{% endifnotequal %}{% endfor %}</td>
+  <td class="clsleft"><b>Coteachers (<a href="mailto:{{ cls.emailcode }}-teachers@{{ email_host_sender }}">email</a>): </b></td>
+  <td class="clsmiddle" colspan="5">{% for t in cls.get_teachers %}{% ifnotequal t user %}{{ t.first_name }} {{ t.last_name }}{% if not forloop.last %}, {% endif %}{% endifnotequal %}{% endfor %}</td>
```

Comment:
The multiple if's in this line generate a slight problem. If it's not the last teacher, it puts a comma, but then when it goes to the next teacher and it's equal to the user, it passes. So if the user happens to be last in the list then you end up with a trailing comma